### ISSUE TITLE
Questions about using SWAT

### ISSUE THREAD

**Issue by fwqwqwqwqwq** [AamirRaihan/SWAT#5]
I want to use SWAT in my FL frame using SWAT's sparsity training，what should I do?

**Comment by AamirRaihan** [comment]
If the FL frame is built on top of Pytorch framework then SWAT can be used directly by installing and importing the mypkg and then importing the using the customlayes such as *custom_layers.custom_conv.CustomConv2d*, *custom_layers.custom_linear.CustomLinear* and *custom_layers.custom_batchnorm.CustomBatchNorm2d* instead of using the pytorch default 'nn.Conv2d', 'nn.Linear',  'nn.BatchNorm2d' respectively. Some examples can be found [here.](https://github.com/AamirRaihan/SWAT/tree/main/SWAT-code/cifar10-100-code/models)
